### ISSUE TITLE
Remove unused accessibility types

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#166]
There are a few unused accessibility types now that we have generalized AccessibleNode.  They should be removed.  We should also determine if other files under accessibility/ can be removed and replaced with the new generalized way of adding PDOM content.

**Comment by jessegreenberg** [comment]
This will include 
- [ ] AccessibleABSwitchNode (rewrite to use AccessibleNode)
- [ ] AccessibleBalloonNode (should be deleted)
- [ ] AccessibleButtonNode (should be deleted, replaced by AccessibleNode)
- [ ] AccessibleDescriptionNode (should be deleted, replaced by AccessibleNode options)
- [ ] AccessibleDivNode (Should be deleted, replaced by AccessibleNode parentContainer option)
- [ ] AccessibleHeadingNode (should be deleted, replaced by AccessibleNode options)
- [ ] AccessibleLegendNode (unclear why this is checked in)
- [ ] AccessibleRadioButtonGroupContent (unused, unclear why checked in)
- [ ] KeyboardHelpDialog (Should be rewritten to use AccessibleNode)

**Comment by jobara** [comment]
@jessegreenberg can any of this generalization be shared with the John Tranvoltage?

**Comment by jessegreenberg** [comment]
> @jessegreenberg can any of this generalization be shared with the John Tranvoltage?

Absolutely, thanks @jobara.  I would like to clean up the common types before committing to common code, but once they are there, it would be great to use in john-travoltage.

**Comment by jessegreenberg** [comment]
Cleanup completed in the above commits.  Closing.
